customScripts/expansion.gd

```diff
@@ -2615,14 +2615,14 @@ func dailyTownEvents():
 		if !currenttown.townhall.fines.empty():
 			if currenttown.townhall.autopay_fines == true:
 				for fine in currenttown.townhall.fines:
-					gold_lost -= int(fine[1])
+					gold_lost -= int(currenttown.townhall.fines[fine][1])
```

Comment:
This entire file appears to be a revert and undoes various fixes.

Reply:
checked out from main for next commit